javascript/2020-11-09-jwt.md

```diff
@@ -0,0 +1,5 @@
+lgtm,codescanning
+* Libraries implementing JWT have been modeled. 
```

Comment:
```suggestion
* The security queries now track taint through JWT decoding, and warns about hard-coded JWT signing keys.
```
I think we should try to be a little less opaque in the these change notes.

It should also be moved to the `javascript/change-notes/` directory.